README.md

```diff
@@ -138,14 +139,23 @@ The main use-case is for JWK where your public key is in a JSON format like so:
    "n":"yRE6rHuNR0QbHO3H3Kt2pOKGVhQqGZXInOduQNxXzuKlvQTLUTv4l4sggh5_CYYi_cvI-SXVT9kPWSKXxJXBXd_4LkvcPuUakBoAkfh-eiFVMh2VrUyWyj3MFl0HTVF9KwRXLAcwkREiS3npThHRyIxuy0ZMeZfxVL5arMhw1SRELB8HoGfG_AtH89BIE9jDBHZ9dLelK9a184zAf8LwoPLxvJb3Il5nncqPcSfKDDodMFBIMc4lQzDKL5gvmiXLXB1AGLm8KBjfE8s3L5xqi-yUod-j8MtvIj812dkS4QMiRVN_by2h3ZY8LYVGrqZXZTcgn2ujn8uKjXLZVD5TdQ"
 }
 ```
+The JWK values can usually be found by looking up the OIDC configuration of the token issuer:
+
+1. append `/.well-known/openid-configuration` to the value of `iss` field from the token and download the JSON file from that URL
+2. look up the URL for `jwks_uri` field in the downloaded document
+3. download JWKS (a JWK set) from `jwks_uri` and use the record with `kid` matching `kid` in the token's header
```

Comment:
As mentioned before I don't think I would include these in the README, it's more likely to confuse people imo. Eg I've used JWKs before but never did any of this.

Reply:
@Keats , Vincent, sorry, I misunderstood your earlier comment! Adding this part was the main driver for the PR. It's aimed at devs who work with this kind of stuff for the first time or not often enough to remember all the ins and outs. This method is not dependent on the Auth provider. If you know an easier way of getting the keys I'd love to know it :)

There is no point applying the rest of the changes if this one is out. I'll kill the PR. Sorry for wasting your time.

Reply:
So for example for my usage of JWKs, I do a HTTP request to my oauth server `.well-known/jwks.json` which gives me something like https://github.com/Keats/jsonwebtoken/blob/master/examples/auth0.rs#L8
Basically https://github.com/Keats/jsonwebtoken/blob/master/examples/auth0.rs is an example of my usage of JWKs. I've never really seen or used the steps from your PR, that's why I find it a bit confusing